src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs

```diff
@@ -259,15 +260,19 @@ public class EntityConverter {
         var entityInfo = GetEntityInfo<T>();
         var parameters =
             entityInfo.properties.Keys.Select(k => GetFieldValue(entityInfo, k, entity)).ToArray();
-
-        var entityRecord = (T)entityInfo.constructor.Invoke(parameters);
-
-        if (entity.ETag != _emptyETag) {
-            entityRecord.ETag = entity.ETag;
-        }
-        entityRecord.TimeStamp = entity.Timestamp;
-
-        return entityRecord;
+        try {
+            var entityRecord = (T)entityInfo.constructor.Invoke(parameters);
+            if (entity.ETag != _emptyETag) {
+                entityRecord.ETag = entity.ETag;
+            }
+            entityRecord.TimeStamp = entity.Timestamp;
+            return entityRecord;
+
+        } catch (Exception) {
```

Comment:
can you also add the content of the exception to the message

Reply:
Yes!